public/index.html

```diff
@@ -39,5 +41,15 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
-  </body>
-</html>
+</body>
+
+<script>
+  // Internet Explorer 6-11
+  const isIE = /*@cc_on!@*/false || !!document.documentMode;
+
+  if (isIE) {
+    document.getElementById("root").innerHTML = "Internet Explorer is not supported. Download Chrome/Opera/Firefox.";
```

Comment:
nit: Please list Firefox first 😄

Reply:
Done. :) I added a separate commit with larger font for the message (you can possibly skip this commit if it's unnecessary).
Additional idea for eg adding browser links in this message (users are lazy). What do you think?

Reply:
I'd prefer to keep the responsibility of this website small -- we'll alert the user but I don't think it's worth going beyond.